en/fieldwork/digitize.rst

```diff
@@ -64,24 +64,25 @@ To edit, tap on **Menu button -> pencil.**
 
 You can edit an existing feature by moving, adding or deleting the vertex and also spliting. 
 
+Split tool
+----------
+
+It is possible to split an existing geometry by drawing a line where it should be split.
+
 Vertex tool
 -----------
 
 It is possible to move and delete vertices of geometries and add new ones.
 
-.. container:: clearer text-left
+demonstration
+-------------
 
-  .. figure:: ../images/edit_geom_feature.gif 
-     :width: 300px
-     :alt: edit_geom_feature
+A video demonstration of some edition possibilities:
```

Comment:
Is this for splitting, vertex editing or all of it?

Reply:
@lucienicolier did you see this question=

Reply:
@m-kuhn  no I didn't see. There is vertex, spilt, and ring on this video

Reply:
Thanks, nice !

I think "editing" is a more appropriate term than edition.

Reply:
done